### PR TITLE
Can't pop back on Web when reload

### PR DESCRIPTION
**Pull request by ManhNTX**
https://user-images.githubusercontent.com/107173849/190536568-4c3bed9b-0875-4e66-92b4-28afd166e016.mov

https://user-images.githubusercontent.com/107173849/190557115-84f9368c-b922-4af3-ac26-3a9c29496d4b.mov

### CODE REVIEW

**Comment by dab246** [comment]
@ManhNTX  Please record demo video on Mobile/Tablet

**Comment by ManhNTX** [comment]
> @ManhNTX Please record demo video on Mobile/Tablet

done

File: lib/features/identity_creator/presentation/identity_creator_controller.dart [hunk 1]
```diff
@@ -67,14 +78,23 @@ class IdentityCreatorController extends BaseController {
...
     _getAllIdentities();
-    if (actionType.value == IdentityActionType.edit && identity != null) {
-      _setUpValueFromIdentity();
-    }
+    _initWorker();
     super.onReady();
   }
 
+  void _initWorker() {
+    identityCreatorIsActiveWorker = ever(_accountDashBoardController.identityCreatorIsActive, (identityCreatorIsActive) {
```
Comment (dab246) [inline lib/features/identity_creator/presentation/identity_creator_controller.dart:88]: Should `dispose` of it when close view

File: lib/features/rules_filter_creator/presentation/rules_filter_creator_controller.dart [hunk 2]
```diff
@@ -46,20 +49,37 @@ class RulesFilterCreatorController extends BaseMailboxController {
...
-  TMailRule? _currentTMailRule;
 
   RulesFilterCreatorController(
     this._verifyNameInteractor,
     this._getAllMailboxInteractor,
     treeBuilder
   ) : super(treeBuilder);
 
+  void _initWorker() {
+    rulesFilterCreatorIsActiveWorker = ever(_accountDashBoardController.rulesFilterCreatorIsActive, (rulesFilterCreatorIsActive) {
```
Comment (dab246) [inline lib/features/rules_filter_creator/presentation/rules_filter_creator_controller.dart:73]: same

**Review by hoangdat — APPROVED** [review]

**Review by dab246 — APPROVED** [review]

**Pull request merged**